v0.1/core/annotation.schema.json

```diff
@@ -15,49 +22,41 @@
-      "description": "How was this annotation determined."
-    },
     "criteria": {
       "oneOf": [{
-        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json/PROTOCOL"
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json/PROTOCOL"
       }, {
-        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json/PROTOCOL_EXECUTION"
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json/PROTOCOL_EXECUTION"
       }],    
       "description": "criteria"
     },
-    "inspiredBy": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE",
-      "description": "inspiredBy"
+    "criteriaType": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
+      "description": "How was this annotation determined."
     },
-    "annotationFile": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/ANNOTATION_FILE",
-      "description": "annotationFile"
+    "definingTerm": {
+      "$ref": "https://schema.hbp.eu/sands/core/anatomicalEntity.schema.json",
+      "description": "Anatomical entity that provides the term for the here specified annotation."
     },
-    "nativeCoordinateSpace": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/COORDINATE_SPACE",
-      "description": "The native coordinate space of this annotation."
+    "inspiredBy": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/IMAGE",
+      "description": "Link to (source) image(s) that inspired the definition of this annotation."
     },
-    "navigationRules": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/NAVIGATION_RULES",
-      "description": "navigationRules"
-    }
     "laterality": {
       "type": "array",
-      "description": "The side of the body or bilateral organ that this anatomical entity relates to.",
+      "description": "The side of the body or bilateral organ that this annotation is defined in.",
       "items": {
         "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM"
       },
       "maxItems": 2
+    },
+    "relatedAtlasTerm": {
+      "$ref": "https://schema.hbp.eu/sands/core/anatomicalEntity.schema.json",
+      "description": "Anatomical entity (term) of a brain atlas annotation to which the here specified annotation is related to."
+    },
```

Comment:
Same as above. Count 0 - n. Define type as array.

Reply:
This needs to be one. It does not make sense to have 0 - n here according to what we discussed. Or do you have a use case in mind?

Reply:
I only worked according to #27 and there it says 0-n. Not saying it should be like that, but I assume you discussed this in one of the meetings I couldn't attend.

Reply:
@aeidi89 What do you think? Any examples for when several would be useful or should it always be one?